forge/forge/op/eval/common.py

```diff
@@ -87,7 +87,9 @@ def create_constant_tensor_from_value(
         tensor[:, :, 0:dim_r, 0:dim_c] = value
     else:
         if tensor_c == 0 and tensor_r == 0:
-            tensor = torch.tensor(value, dtype=dtype)
+            # Unsqueezing to make this a 1d tensor.
+            # Currently, the runtime expects a 1d tensor for scalar values.
```

Comment:
Should this be by design for the runtime? If no, we should track this as separate issues on MLIR side